src/app/containers/MainContent/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import headline from '../Headline';
+import text from '../Text';
+import mainContentModelPropTypes from '../../models/propTypes/mainContent';
+import MainContent from '../../components/MainContent';
+
+// Inlined as this is a temporary component
+const BlockString = props => {
+  const stringProps = JSON.stringify(props);
+  return <p>{stringProps}</p>;
+};
+
+const Blocks = {
+  headline,
+  text,
+};
+
+const render = blocks =>
+  blocks.map(block => {
+    const { type, blockId, model } = block;
+
+    const Block = Blocks[type] || BlockString;
+
+    return <Block key={blockId} {...model} />;
```

Comment:
This will also need to pass type down into the downstream components, or that could be done in Paul's PR - https://github.com/bbc/simorgh/issues/298

Reply:
Good point. Will be done in https://github.com/bbc/simorgh/pull/359.